plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -57,10 +57,20 @@ export function createPublishGithubAction(options: {
     requireCodeOwnerReviews?: boolean;
     requiredStatusCheckContexts?: string[];
     repoVisibility?: 'private' | 'internal' | 'public';
-    collaborators?: Array<{
-      username: string;
-      access: 'pull' | 'push' | 'admin' | 'maintain' | 'triage';
-    }>;
+    collaborators?: Array<
+      | {
+          user: string;
+          access: 'pull' | 'push' | 'admin' | 'maintain' | 'triage';
+        }
+      | {
+          team: string;
+          access: 'pull' | 'push' | 'admin' | 'maintain' | 'triage';
+        }
+      | {
+          username: string;
```

Comment:
Think that you can also add `/** @deprecated */` to this too and it will possibly show up in the editor 🤞